src/main/webapp/project/NewProjectGeneralData.js

```diff
@@ -194,30 +203,30 @@ export const NewProjectGeneralData = hh(class NewProjectGeneralData extends Comp
           })
         ]),
 
-        Panel({ title: "Principal Investigator ", moreInfo: "(if applicable)" }, [
-          MultiSelect({
-            id: "pi_select",
-            label: "Broad PI",
-            isDisabled: false,
-            loadOptions: this.loadUsersOptions,
-            handleChange: this.handlePIChange,
-            value: this.state.formData.piName,
-            placeholder: "Start typing the PI Name",
-            isMulti: false,
-            edit: false
-          }),
-          MultiSelect({
-            id: "inputProjectManager",
-            label: "Broad Project Manager",
-            isDisabled: false,
-            loadOptions: this.loadUsersOptions,
-            handleChange: this.handleProjectManagerChange,
-            value: this.state.formData.projectManager,
-            placeholder: "Start typing the Project Manager Name",
-            isMulti: false,
-            edit: false
-          }),
-        ]),
+          Panel({ title: "Principal Investigator ", moreInfo: "(if applicable)" }, [
+            MultiSelect({
+              id: "pi_select",
+              label: "Broad PI",
+              isDisabled: false,
+              loadOptions: this.loadUsersOptions,
+              handleChange: this.handlePIChange,
+              value: this.state.formData.piName,
+              placeholder: "Start typing the PI Name",
+              isMulti: false,
+              edit: false
+            }),
+            MultiSelect({
+              id: "inputProjectManager",
+              label: "Broad Project Manager",
+              isDisabled: false,
+              loadOptions: this.loadUsersOptions,
+              handleChange: this.handleProjectManagerChange,
+              value: this.state.formData.projectManager,
+              placeholder: "Start typing the Project Manager Name",
+              isMulti: false,
+              edit: false
+            }),
+          ]),
```

Comment:
This looks like a lot of white space changes with no purpose. I have seen a lot of this in different PRs. I think the team needs to agree on a standard and set up your IDEs so you're not overwriting each others' changes for no reason. The volume of these kinds of changes is really too much and just adds noise to everyone's PRs.

Reply:
We're discussing with the team to implement the same linting template so this stops to happen.

Reply:
There are lots of examples out there ... 
* Grails plugin https://grails.org/plugin/jslint
* Gradle plugin: https://github.com/nebula-plugins/gradle-lint-plugin
* Intellij plugin: https://www.jetbrains.com/help/idea/jslint.html